components/home/__tests__/homepage-whatsapp-cta.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { HomePageWhatsAppCTA } from "../homepage-whatsapp-cta";
+
+import { render } from "@testing-library/react";
+
+describe("HomePageContributing", () => {
+  it("renders correctly", () => {
+    const { container } = render(<HomePageWhatsAppCTA />);
+
+    expect(container.firstChild).toMatchInlineSnapshot(`
+      <section
+        class="bg-white overflow-hidden px-4 py-6 space-y-4 text-center"
+      >
+        <h2
+          class="font-semibold"
+        >
+          Masih belum dapat info yang kamu cari?
+        </h2>
+        <a
+          class="flex flex-row px-4 py-2 text-sm rounded-full items-center justify-center border border-transparent font-medium shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-75 text-white bg-[#25d366] focus:ring-[#25d366]"
+          href="https://bit.ly/halowarga"
```

Comment:
The snapshot test is good for asserting the styling changes.
Nevertheless, I think it would be helpful if we add another test assertion this `href` property to signal the developers that it's something that matters in this component.

Reply:
@zainfathoni Alright, let's add that as a task in #200.